src/WallToolPaths.cpp

```diff
@@ -108,12 +206,6 @@ const Polygons& WallToolPaths::getOutline() const
     return outline;
```

Comment:
`getOutline()` is never used remove it

Reply:
Indeed, apparently you're right. I just left this function there untouched, assuming it was necessary.

Removed here: 0142d2417fb09b5064d66653162d7ea8bc9163b1